libnavigator/src/main/java/com/mapbox/navigation/navigator/ActiveGuidanceOptionsMapper.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.navigation.navigator
+
+import com.mapbox.api.directions.v5.DirectionsCriteria
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.navigator.ActiveGuidanceGeometryEncoding
+import com.mapbox.navigator.ActiveGuidanceMode
+import com.mapbox.navigator.ActiveGuidanceOptions
+
+internal class ActiveGuidanceOptionsMapper {
+
+    fun mapFrom(directionsRoute: DirectionsRoute?): ActiveGuidanceOptions {
+        if (directionsRoute == null) {
```

Comment:
nit: could delete this since mapToActiveGuidanceMode and mapToActiveGuidanceGeometry are both handling null